src/main/connector/system/WindowsService/WindowsService.yaml

```diff
@@ -33,19 +33,21 @@ monitors:
   service:
     simple:
       sources:
-        # Name;ProcessId;StartMode;State
+        # Name;State;StartMode;DisplayName;StartName;PathName;ProcessId
         serviceInformation:
           type: wmi
           namespace: root\CIMv2
           query: |
             SELECT
               Name,
               State,
-              Description
+              StartMode,
+              DisplayName,
+              StartName,
+              PathName,
+              ProcessId
             FROM
               Win32_Service
-            WHERE
-              StartMode != 'Disabled'
           computes:
           - type: keepOnlyMatchingLines
             column: 1

```

Comment:
In the LinuxService connector, we use a `regEx`, and in WindowsService, we use a `valueList`.

I would be in favor of using regular expressions for both.

Variable name can remain `serviceNames`, but it would need to be described clearly as a regular expression, with an example like: `serviceNames: cron|PatrolAgent|snmp`.